script/translations/migrate.py

```diff
@@ -169,24 +173,142 @@ def interactive_update():
         print()
 
 
+STATE_REWRITE = {
+    "Off": "[%key:common::state::off%]",
+    "On": "[%key:common::state::on%]",
+    "Unknown": "[%key:common::state::unknown%]",
+    "Unavailable": "[%key:common::state::unavailable%]",
+    "Open": "[%key:common::state::open%]",
+    "Closed": "[%key:common::state::closed%]",
+    "Connected": "[%key:common::state::connected%]",
+    "Disconnected": "[%key:common::state::disconnected%]",
+    "Locked": "[%key:common::state::locked%]",
+    "Unlocked": "[%key:common::state::unlocked%]",
+    "Active": "[%key:common::state::active%]",
+    "active": "[%key:common::state::active%]",
+    "Standby": "[%key:common::state::standby%]",
+    "Idle": "[%key:common::state::idle%]",
+    "idle": "[%key:common::state::idle%]",
+    "Paused": "[%key:common::state::paused%]",
+    "paused": "[%key:common::state::paused%]",
+    "Home": "[%key:common::state::home%]",
+    "Away": "[%key:common::state::not_home%]",
+    "[%key:state::default::off%]": "[%key:common::state::off%]",
+    "[%key:state::default::on%]": "[%key:common::state::on%]",
+    "[%key:state::cover::open%]": "[%key:common::state::open%]",
+    "[%key:state::cover::closed%]": "[%key:common::state::closed%]",
+    "[%key:state::lock::locked%]": "[%key:common::state::locked%]",
+    "[%key:state::lock::unlocked%]": "[%key:common::state::unlocked%]",
+}
+SKIP_DOMAIN = {"default", "scene"}
+STATES_WITH_DEV_CLASS = {"binary_sensor", "zwave"}
```

Comment:
What about sensors?

Reply:
This is about translations in the frontend that have device classes.